FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml

```diff
@@ -238,7 +238,7 @@
                                     Text="{x:Bind Arguments, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
                                 <ComboBox
                                     Width="150"
-                                    Margin="0,24,0,0"
+                                    Margin="0,24,0,16"
```

Comment:
This seems to be an accidental/unnecessary/unrelated change?

Reply:
Can you look at this change again? It seems to be causing this gap ⬇️

![gap preview](https://user-images.githubusercontent.com/3742559/52616631-d7fff900-2ea1-11e9-8e06-daad28d8579f.png)